homeassistant/components/plex/update.py

```diff
@@ -0,0 +1,81 @@
+"""Representation of Plex updates."""
+import logging
+from typing import Any
+
+from plexapi.exceptions import PlexApiException
+import plexapi.server
+import requests.exceptions
+
+from homeassistant.components.update import UpdateEntity, UpdateEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import CONF_SERVER_IDENTIFIER
+from .helpers import get_plex_server
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Plex media_player from a config entry."""
+    server_id = config_entry.data[CONF_SERVER_IDENTIFIER]
+    server = get_plex_server(hass, server_id)
+    plex_server = server.plex_server
+    async_add_entities([PlexUpdate(plex_server)], update_before_add=True)
+
+
+class PlexUpdate(UpdateEntity):
+    """Representation of a Plex server update entity."""
+
+    _attr_entity_category = EntityCategory.CONFIG
+    _attr_supported_features = (
+        UpdateEntityFeature.INSTALL | UpdateEntityFeature.RELEASE_NOTES
+    )
```

Comment:
Not every plex instance can handle the install feature. Therefore, the `UpdateEntityFeature.INSTALL` flag should only be added if that is the case.